src/EventSubscriber/AbstractGDPRSubscriber.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace App\EventSubscriber;
+
+use App\Entity\User;
+use App\Security\HelperV2\Oauth2Helper;
+use App\ServiceV2\Traits\UserAwareTrait;
+use League\Bundle\OAuth2ServerBundle\Security\Authentication\Token\OAuth2Token;
+use Symfony\Bundle\SecurityBundle\Security;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Component\Routing\RouterInterface;
+use Symfony\Component\Security\Core\Authentication\Token\Storage\TokenStorageInterface;
+
+abstract class AbstractGDPRSubscriber implements EventSubscriberInterface
```

Comment:
I'm not sure that the name is right. I understand that we use it in GDPR issues context, but when you look at the class itself, it does not have anything to do with GDPR